### PR TITLE
Split out example content and updated README

### PR DESCRIPTION
**Pull request by sabre1041**
Split out the example content destined for its own repository and updated the contents of the README

@jawnsy @alberttwong please review

### CODE REVIEW

**Comment by sabre1041** [comment]
Travis build failed due to example application removal

File: README.md [hunk 1]
```diff
@@ -6,141 +6,55 @@ Red Hat OpenShift Source-to-Image template for ASP.NET applications
...
+source code.
 
-High level process
-
-1. Create the OpenShift ASP.NET base image (creates docker image 'aspapp')
-2. Merge source code using the S2I tool into the OpenShift image (creates docker image 'aspnet-app')
-3. Run the new OpenShift ASP.NET base+code image
-
-For more information about using these images with OpenShift, please see
+For more information about using the Source to Image with OpenShift, please see
```
Comment (jawnsy) [inline README.md]: _using the Source to Image_

Remove "the"

**Comment by sabre1041** [comment]
@jawnsy changes have been updated

**Comment by jawnsy** [comment]
Changes merged, thanks!

**Pull request closed without merging**